src/components/AnnotationContentOverlay/AnnotationContentOverlay.js

```diff
@@ -55,6 +64,20 @@ const AnnotationContentOverlay = () => {
   const contents = annotation?.getContents();
   const numberOfReplies = annotation?.getReplies().length;
 
+  if (annotation && isUsingCustomHandler) {
```

Comment:
Hmm I think we still want to have the `isDisabled` and `isMobileDevice` check for the custom tooltip. What do you think?

Also I think the code would look a bit more better if we can put the `CustomElement` inside of the `<div>` below. For example:
```javascript
// may need to pass contents to <DefaultOverlay>, or maybe just pass `annotation`
return isDisabled || isMobileDevice || !contents ? null : (
  <div
    className="Overlay AnnotationContentOverlay"
    data-element="annotationContentOverlay"
    style={{ ...overlayPosition }}
  >
    {isUsingCustomHandler ? <CustomElement /> : <DefaultOverlay ... />}
  </div>
);

const DefaultOverlay = (props) => (
  <>
    <div className="author">{core.getDisplayAuthor(annotation)}</div>
    <div className="contents">
      {contents.length > MAX_CHARACTERS
        ? `${contents.slice(0, MAX_CHARACTERS)}...`
        : contents}
    </div>
    {numberOfReplies > 0 && (
      <div className="replies">
        {t('message.annotationReplyCount', { count: numberOfReplies })}
      </div>
    )}
  </>
)
```

Reply:
@ZhijieZhang Refactored things a bit, little different from your proposal due to the `contents` part of the conditional rendering, but I think the code is a bit cleaner.